react-commenting.js

```diff
@@ -111,14 +113,14 @@ class Board extends Component{
 class Square extends Component{
 
   handleSquareClick = () => {
-    // 12) Here:
+    // 12) Here: gives the abiliy to click on a squaret
     this.props.gamePlay(this.props.index)
```

Comment:
It does, it calls the method gamePlay and passes the index of the square that was clicked to the Board component.